stdlib/builtins.pyi

```diff
@@ -341,16 +339,15 @@ class complex:
     def __mul__(self, __x: complex) -> complex: ...
     def __pow__(self, __x: complex, mod: None = ...) -> complex: ...
     def __truediv__(self, __x: complex) -> complex: ...
+    def __eq__(self, __x: object) -> bool: ...
+    def __ne__(self, __x: object) -> bool: ...
     def __radd__(self, __x: complex) -> complex: ...
     def __rsub__(self, __x: complex) -> complex: ...
     def __rmul__(self, __x: complex) -> complex: ...
     def __rpow__(self, __x: complex, __mod: None = ...) -> complex: ...
     def __rtruediv__(self, __x: complex) -> complex: ...
```

Comment:
```suggestion
    def __radd__(self, __x: complex) -> complex: ...
    def __rsub__(self, __x: complex) -> complex: ...
    def __rmul__(self, __x: complex) -> complex: ...
    def __rpow__(self, __x: complex, __mod: None = ...) -> complex: ...
    def __rtruediv__(self, __x: complex) -> complex: ...
    def __eq__(self, __x: object) -> bool: ...
    def __ne__(self, __x: object) -> bool: ...
```
Let's keep it where it was to reduce git blame noise

Reply:
Yeah I've already reverted that, my mistake.